hr_holidays_leave_auto_approve/README.rst

```diff
@@ -0,0 +1,40 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Thanks for the informations. The readme file should now be consistent with the convention.